packages/mangrove/src/eth.ts

```diff
@@ -1,12 +1,7 @@
-export type Address = string;
+import { TxRef } from "@proximaone/stream-schema-base";
+export { Hash, Address } from "@proximaone/stream-schema-base";
+
 export type UInt = string;
-export type Hash = string;
-export type BlockNumber = number;
 export type Int = string;
 
-export type TransactionRef = {
-  blockNumber: BlockNumber;
-  blockHash: Hash;
-  txHash: Hash;
-  from: Address;
-};
+export type TransactionRef = TxRef;
```

Comment:
Preserved here for compatibility. However, in the new code we should use `TxRef` directly.